index.html

```diff
@@ -14,6 +14,109 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <section class="left-panel">
+
+    <h1>David Lam</h1> 
+    <h3>Web Developer</h3>
+
+    <h2>Personal Info</h2>
+    <h4>Address</h4>
+    <p>7911 Santa Barbara Ave<br/>
+    Stanton, CA<br/>
+    90680</p>
+    <h4>Phone</h4>
+    <P>*** *** ****</P>
+    <h4>E-mail</h4>
+    <p><span>DavidLamSgn</span><span>@gmail.com</span></p>
+    <h4>LinkedIn</h4>
+    <p>linkedin.com/DavidLam</p>
```

Comment:
Consider turning this into a link. Maybe something like:
`<a href="linkedin.com/DavidLam" target="_blank"><p>linkedin.com/DavidLam</p></a>`